doc/WebRTC.xml

```diff
@@ -672,6 +694,39 @@ Server -> Device: { "method": "trickle", "params": {"session": "s1", "candidate"
       <para>For two-way audio it's recommended to use echo cancellation to avoid feedback
         loops.</para>
     </section>
+    <section xml:id="section_data_channels">
+      <title>Data channels</title>
+        <section xml:id="section_data_channels_enable">
+          <title>Enabling data channels</title>
+            <para>If the device supports WebRTC data channels, it shall signal this in the SDP offer 
+              of the invite method. If the client supports data channels and intends to use them during 
+              the WebRTC session, it shall signal this in the SDP answer.</para>
+            <para>ONVIF defines data channel subprotocol names starting with the “vnd.onvif” prefix. 
+              Usage of this prefix is restricted to ONVIF-defined subprotocols only.</para>
+            <para>Before creating and using any data channel subprotocol, the device and client need 

```

Comment:
Before creation and usage of any data channel subprotocol, the device and client shall exchange a list of allowed data channel subprotocols in the invite method.

Reply:
Rephrased as suggested